docs/kafka/rhoas-cli-getting-started-kafka/README.adoc

```diff
@@ -105,6 +106,9 @@ This example creates a Kafka instance called `my-kafka`.
 $ rhoas kafka create --name my-kafka
 ----
 
+CLI uses service contexts to group service instances against which commands are executed.
```

Comment:
@wtrocki 
Updated the Kafka guide, do you think this suffices for referencing `contexts` in rest of the guide, we can add link to context-getting-started-doc once it is ready.

Reply:
To make sure I understand. Plan is to create mini guide for context specifically and reference it in both kafka and services registry. If that is the plan then +100 :D

Reply:
Yes. There is going to be a guide for context solely.